scripts/setup/setup-certbot

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+  echo "Usage: $0 <domain name> <email>" >&2
+  exit 1
+}
+
+if [ "$EUID" -ne 0 ]; then
+    echo "Error: This script must be run as root" >&2
+    exit 1
+fi
+
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+DOMAIN="$1"
+EMAIL="$2"
+
+# For reference https://certbot.eff.org/all-instructions/#ubuntu-14-04-trusty-nginx
+apt-get update
+apt-get install -y software-properties-common
+add-apt-repository -y ppa:certbot/certbot
+apt-get update
+apt-get install -y python-certbot-nginx
```

Comment:
Have you tested this on a production server?  I seem to recall having problems with the certbot PPA installing a newer version of openssl or something that broke our virtualenvs.  We might need to figure out another way to install certbot.

Reply:
Nope, the DO droplet, I am not touching, to avoid breaking things since it is running several scripts (tunnels, mirrors, ...). Also, @feorlen is busy testing in her own server nowadays.
I am only able to properly dev locally only if the prod Dockerfile has been finalized.

Reply:
If there is problem, then the upstream documentation, https://certbot.eff.org/all-instructions/#ubuntu-14-04-trusty-nginx must be fixed.

Reply:
Ok, I will just disable the `--nginx`.

Reply:
and leave `certbot renew` to the user